qemu_v8.mk

```diff
@@ -152,6 +154,8 @@ linux-defconfig: $(LINUX_PATH)/.config
 LINUX_COMMON_FLAGS += ARCH=arm64 Image
 
 linux: linux-common
+	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS) modules
+	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS) modules INSTALL_MOD_STRIP=1 INSTALL_MOD_PATH=$(MODULE_OUTPUT) modules_install
```

Comment:
- `make modules` twice?
- This will significantly increase the build time for everyone, including people who don't need any module (the `optee_test` suite runs fine without any module). Could we find a way to address this? Perhaps add a new target?

Reply:
Thanks for your review :) In v2 I have removed the duplicated modules, and split it into a separate linux-modules target as you suggested.